### PR TITLE
Add documentation for updatedProduct

### PR DESCRIPTION
**Pull request by saturnxxi**
updateProduct is fired before the combination update starts. updatedProduct after that

### CODE REVIEW

**Comment by matks** [comment]
Hi @saturnxxi thanks for the catch 😉 

However if I'm not wrong, there is **both** `updateProduct` and `updatedProduct` events so we should not replace what is in the doc but **add** what is missing

@zalexki Can you confirm both `updateProduct` and `updatedProduct` events exist ?

**Comment by zalexki** [comment]
Hey guys, hum i don't know much about this but i can see both events are registered and emitted the in core.js and it's map file. 

For the rest ¯\_(ツ)_/¯ 

I guess it make sens but i didn't tested it.

**Comment by matks** [comment]
@Quetzacoalt91 help 😅 

Can you confirm both updateProduct and updatedProduct events exist ?

File: src/content/1.7/themes/reference/javascript-events/_index.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Event Name            | Description
  `changedCheckoutStep` | Each checkout step **submission** will fire this event.
  `updateProductList`  | On every product list page (category, search results, pricedrop and so on), the list is updated via ajax calls if you change filters or sorting options. Each time the DOM is reloaded with new product list, this event is triggered.
  `clickQuickView`     | If your theme handles it, this event will be trigged when use click on the quickview link.
- `updateProduct`      | On the product page, selecting a new combination will reload the DOM via ajax calls. After the update, this event is fired.
```
Comment (matks) [inline src/content/1.7/themes/reference/javascript-events/_index.md]: `updateProduct` event exists too, so please add `updatedProduct` but also keep `updateProduct` 😄

**Comment by matks** [comment]
> @matks I confirm you that both exists.

Thanks ! Then cannot accept his PR as it is

**Comment by matks** [comment]
I did the change. Thank you @saturnxxi

**Pull request merged**